.idea/vcs.xml

```diff
@@ -22,4 +22,4 @@
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/docs/themes/book" vcs="Git" />
   </component>
-</project>
+</project>
```

Comment:
Maybe we shouldn't change vcs.xml file?

Reply:
Yes,I don't know when this file changed and I still committed it...